builtin/unsafeconv.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 The Scriggo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package builtin
+
+import (
+	"github.com/open2b/scriggo/native"
+)
+
+// Unsafeconv provides a package that contains functions to make unsafe conversions between string values and native
```

Comment:
I would use `implements` instead of `contains`.

Reply:
Limit the length of the row to 80 characters.

Reply:
changed in "review: wrap comment"